### PR TITLE
RATEST-149: Resurrect checkModule test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-149)

Description -> Resurrect checkModule test

### CODE REVIEW

**Comment by kdaud** [comment]
@sherrif10 take a look at this commit so that we have it sorted into master soon

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+    @Test
+    @Category(BuildTests.class)
+    public void checkModulesTest() {
         AdministrationPage administrationPage = homePage.goToAdministration();
         ModulesPage modulesPage = administrationPage.goToManageModulesPage();
         // Get the modulesListing <div>, which contains the table of modules.
         WebElement moduleListing = modulesPage.findElementById("moduleListing");
         // Grab all the <input> elements from the first column of the table.
-        List<WebElement> firstColumn = moduleListing.findElements(By.xpath("table/tbody/tr/td[1]/input"));
+        List<WebElement> firstColumn = moduleListing.findElements(By.cssSelector("#moduleListing table tbody td"));
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:27]:
```suggestion
Why not using this as an id , something like 
        List<WebElement> firstColumn = moduleListing.findElements(By.cssSelector("#moduleListing"));       
```

**Comment by sherrif10** [comment]
Hey @kdaud  , this ticket is being assigned to @ivan nswookwa  here https://github.com/openmrs/openmrs-distro-referenceapplication/pull/518  .Did you conset with him first

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java (hunk 1 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:27]: I gave on attempt using id locator but test fail sand the one element locator I found out passing the test was by using `cssSelector()`

**Comment by kdaud** [comment]
> Did you consent with him first

We had a sync yesterday with @insookwa when fixing [RATEST-90](https://issues.openmrs.org/browse/RATEST-91) together with him which I believe was able to resurrect the test and hoping to see an updated PR for that test and was very positive to give a helping hand for the [RATEST-149](https://issues.openmrs.org/browse/RATEST-149) and [RATEST-91](https://issues.openmrs.org/browse/RATEST-91). 

We want to finish up all the remaining selenium legacy tests to make sure we have 100% coverage for them, so am currently giving a hand for the remaining tests to be clearly.
cc: @dkayiwa

**Comment by sherrif10** [comment]
Then i think its good to un assign him from that ticket and assign yourself. Thanks

**Comment by kdaud** [comment]
@sherrif10 any other recommendation ?

**Review by sherrif10 — APPROVED** [review]
Ready to be merged   LGTM cc @dkayiwa  , Thanks @kdaud

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void checkModules() throws Exception {
+    @Test
+    @Category(BuildTests.class)
+    public void checkModulesTest() {
```
Comment (dkayiwa) [inline ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:21]: Is there any reason for changing the original method name?

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:21]: Selenium legacy ui tests method naming follow a convention of naming the test method the `testClassName` except beginning with lower Case letter as embraced by method naming  in java programming !

That side, already existing test method names gives a hint why changing the original method name to checkModuleTest() was done. 
Below is are four samples of existing method naming !
[addUserTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java#L31) for AddUserTest class
[endVisitTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java#L36) for EndVisitTest class
[searcActiveVisitTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java#L36) for SearchActiveTest class

[xssOnPhoneNumberFieldTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java#L36)

**Comment by sherrif10** [comment]
Hello @kdaud  this ticket have issues, Try re running it locally it throws null pointer Exceptrions to fix, Am wondering why it keeps passing on ci

**Comment by kdaud** [comment]
> this ticket have issues, Try re running it locally

@sherrif10 crosscheck the build summary [here](https://pastebin.com/9Q2i4PSJ) have obtained from the assignment.

**Pull request merged**